packages/core/src/accessanalyzer/vue/iamPolicyChecks.vue

```diff
@@ -225,19 +242,24 @@ const client = WebviewClientFactory.create<IamPolicyChecksWebview>()
 export default defineComponent({
     mixins: [saveData],
     data: () => ({
-        documentType: 'JSON Policy Language',
+        documentType: 'CloudFormation',
         policyType: 'Identity',
```

Comment:
worth adding comments and/or renaming it to indicate this is used for validatePolicy API